### DISCUSSION TITLE
Problems trying to train custom or uploaded models

### DISCUSSION THREAD

**Discussion by ClaraSanders** [TheLastBen/fast-stable-diffusion#294] [Q&A]
Hey, I recently tested the new fast dreambooth with multiple subjects and it is fantastic. However, I have a big issue that I can't solve. I asked for help on reddit's stablediffusion and on discord with no luck. Fast dreambooth overfits all my models so they are unusable unless I do the training the first time, downloading sd 1.5 from hugging face and letting the repo generate the ckpt from it. 

In other words, if I try to train a custom model loading the custom model and pasting the link on the CKPT_Path cell or I resume a previously created session I overfit the model no matter the steps: it happens with 6000 steps and with 200, it happens too with one subject and with three subjects. What I am doing wrong? Why it only works for me the first time I train a model based on sd 1.5 only?

**Comment by TheLastBen** [comment]
you are probably using the old method, the new method reduces the overfitting greatly and the models are retrainable

**Reply by ClaraSanders** [reply]
How I am supposedly using the old method if I keep the Yes in the Use_New_Fast_Method cell and why the models also end up overfit even if I use a brand new model that I uploaded? I'm not trying to be mean.

**Reply by TheLastBen** [reply]
what did you name your instance images ? and how many

**Reply by ClaraSanders** [reply]
I've run eight tests with many different subjects. The last one was 86 anime pics named phxxnxx (1).jpg, phxxnxx (2).jpg, etc with only one subject using waifu 1.3 uploaded by me. I'm sorry for bothering you, it is just it is so weird what happens. EDIT: Also I tried something different. I used a modified version of your repo done by another guy and it created a ckpt with the same size as the uploaded ckpt instead of 3,97 gigs but the model was also overfit. It is like a freaking curse.

**Reply by TheLastBen** [reply]
86 instance pics is too much, try 15 with 1500 steps at first

**Reply by ClaraSanders** [reply]
I'll try it, thanks. And you know what? I'm pissed not at you but at the repo. I'm gonna f*ck*ng record myself doing the process from start to finish, upload the video to YT, and show it to you. I will keep you informed.
